### PR TITLE
Removed console logs

### PR DESCRIPTION
**Pull request by olliecase-green**

### CODE REVIEW

File: knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js [hunk 1]
```diff
@@ -25,12 +25,9 @@ export default function ViewData(props) {
     })();
   }, []);
 
-  console.log(props.location.state);
-  console.log(props.location.state.students[0].user_id);
   function handleClickData(e) {
     setUserID(e.target.value);
     setRedirect(true);
-    console.log("hi");
```
Comment (zaynshah) [inline knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js:33]: Good job at removing the console log.

File: knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js [hunk 2]
```diff
@@ -58,19 +56,21 @@ export default function ViewData(props) {
...
+    if (response === 200) {
+      console.log(response);
+      setFullCohortLOs(
+        await network.getAllTopicsPerCohort(
+          props.location.state.cohortLOs[0].cohort_id
+        )
+      );
+      setLO("");
+      setTopic("");
+    }
```
Comment (zaynshah) [inline knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js:73]: Good bug fix

File: knowledge-challenge-app/src/Components/AdminDashboard/AdminDashboard.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ export default function AdminDashboard(props) {
         {redirect ? (
           <Redirect
             to={{
-              pathname: "/cohort",
+              pathname: "/cohorts",
```
Comment (FahmidulHaquee) [inline knowledge-challenge-app/src/Components/AdminDashboard/AdminDashboard.js:47]: For consistency and stop breakages, nice!

File: knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js [hunk 4]
```diff
@@ -25,12 +25,9 @@ export default function ViewData(props) {
     })();
   }, []);
 
-  console.log(props.location.state);
```
Comment (FahmidulHaquee) [inline knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js:28]: Good job, don't want to confuse the user.

**Review by zaynshah — APPROVED** [review]
Good I cannot see any changes that need to be made.

File: knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js [hunk 5]
```diff
@@ -58,19 +56,21 @@ export default function ViewData(props) {
...
-    setFullCohortLOs(
-      await network.getAllTopicsPerCohort(props.location.state.cohortLOs[0])
-    );
-    setLO("");
-    setTopic("");
+    if (response === 200) {
+      console.log(response);
+      setFullCohortLOs(
+        await network.getAllTopicsPerCohort(
+          props.location.state.cohortLOs[0].cohort_id
```
Comment (FahmidulHaquee) [inline knowledge-challenge-app/src/Components/AdminDashboard/ViewData.js:68]: Good use of error handling!

**Review by FahmidulHaquee — APPROVED** [review]

**Comment by FahmidulHaquee** [comment]
Nice work resolving the LO fetching and re-rendering with Sonali.

**Pull request merged**